source/_docs/clone-site.md

```diff
@@ -20,7 +20,13 @@ After selecting **[Migrate Existing Site](https://dashboard.pantheon.io/sites/mi
 terminus drush <site>.<env> -- ard --destination=code/sites/default/files/<RANDOM_HASH>.tgz
 ```
 
-This operation writes the archive to Pantheon's filesystem in a web accessible location (e.g. `http://env-site-name.pantheonsite.io/sites/default/files/<RANDOM_HASH>.tgz`). Click **Continue Migration** and follow all remaining instructions within the guided migration process.
+This operation writes the archive to Pantheon's filesystem in a web accessible location (e.g. `http://env-site-name.pantheonsite.io/sites/default/files/<RANDOM_HASH>.tgz`).
+<div class="alert alert-danger" role="alert">
+<h4 class="info">Warning</h4>
+<p markdown="1">We reccomend using a random hash for the archive filename for security. A bad actor noticing a site migration underway could attempt to download well-known filename patterns (like `BACKUP` or recent dates). We also advise prompt removal of archive files once you've downloaded them locally.</p>

```

Comment:
Instead of `A bad actor noticing a site migration underway could attempt to download well-known filename patterns (like `BACKUP` or recent dates).`, maybe more generically: `Archive dumps contain sensitive information, so they should not be exposed using guessable filename patterns (like `BACKUP` or recent dates).`

The way you have it is fine, though.

As for ` once you've downloaded them locally.`, this phrase might actually be confusing. The best workflow is to put your archive online in the `files` directory, as described, and then import it into Pantheon by providing the URL to the archive during the migrate process. While it is possible to download the archive locally and upload it to migrate, the size of the archive file can be much larger when downloading by URL. Also, if the user's intention is to upload the archive, then they would be better off creating the archive locally, as that procedure is more secure. (It would also be harder to document, as it would first be necessary to copy the site locally -- a process that could vary slightly, depending on the current hosting provider.)